tensorboard/webapp/metrics/actions/index.ts

```diff
@@ -248,5 +248,10 @@ export const rangeSelectionToggled = createAction(
     affordance?: TimeSelectionToggleAffordance;
   }>()
 );
+
+export const metricsHideEmptyCardsChanged = createAction(
```

Comment:
I think in other places where we are assuming we are just toggling a boolean with the action, we use "Toggled" in the action name instead of "Changed".